.appveyor.yml

```diff
@@ -0,0 +1,28 @@
+build: false
+
+environment:
+  matrix:
+    - PYTHON: "C:\\Miniconda36-x64"
+      PY: 3.7
```

Comment:
This is working on my account. It would be nice if you could enable it here, on the main repo, so we can see the status for each PR issued.

See https://ci.appveyor.com/project/ocefpaf/utide/builds/23342190

Reply:
I _think_ I've done this. I also invited you to my team and made you an admin there, so you should be able to see this project

Reply:
Never used AppVeyor before, so let me know if there is other stuff I need to do

Reply:
Cool. It is active. I'll update to PR with the badge let's see if AppVeyor starts as expected.